opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/Provisioner.java

```diff
@@ -542,7 +544,7 @@ public void doImport(final String url, final String rescanExisting, ProvisionMon
                     resource = new FileSystemResource(file);
                 } else {
                     final String filename = file.getName();
-                    if (filename.contains("%20")) {
+                    if (isNotBlank(filename) && filename.contains("%20")) {
```

Comment:
Isn't this redundant? Something can not be empty and contains an element.

Reply:
Well, just trying to avoid any NPE in case the file name is null

Reply:
`File#getName` is guaranteed to never return `null`.